src/ml_board/frontend/Dashboard/src/worker_socket/MainThreadCallbacks.ts

```diff
@@ -6,7 +6,7 @@ import handleJobStatusData from "./event_handlers/JobStatusHandler";
 
 // Note: I don't like having anything here other than the callbacks but it is what it is :')
 // ========================= variables ============================//
-// TODO: in #145 #147 this has been done with out the need of this, check if still needed (check the TODO drafts)
+// TODO: in #145 #147 this has been done with out the need of this, check if still needed (check #150)
```

Comment:
this kind of things should be normally tracked in issues or PRs and not in the code ;-)

Reply:
okay, I removed it completely